index.html

```diff
@@ -0,0 +1,49 @@
+<html>
+<head>
+<link rel="stylesheet" href="styles.css">
+</head>
```

Comment:
I think you should use the <title> tag in your <head> section because the page's title is very important and helpful to know about the nature and content of that specific page.

Reply:
- [ ] Very nice usage of inline comments. please keep it up while doing reviews for your partners and colleagues.